src/context/ipfs/ipfsnode/IPFSNode.ts

```diff
@@ -38,12 +38,12 @@ class IPFSNode {
 
   /** Start the connection to IPFS (if not connected already) */
   start(): Promise<void> {
-    return this._ipfs.start();
+    return this.ipfs.start();
   }
 
   /** Stop the connection to IPFS */
   async stop(): Promise<void> {
-    return this._ipfs.stop();
+    return this.ipfs.stop();
```

Comment:
I know this is just a linting issue, I just want to note that IPFSNode is only used for development.

What's more, it's not even supported anymore and we'll have to migrate to `helia` eventually